datafusion/sqllogictest/test_files/coalesce.slt

```diff
@@ -260,8 +260,8 @@ select
   arrow_typeof(coalesce(c, arrow_cast('b', 'Dictionary(Int32, Utf8)')))
 from t;
 ----
-a Dictionary(Int32, Utf8)
-b Dictionary(Int32, Utf8)
+a Utf8View
```

Comment:
Dictionary(Int32, Utf8) 

changes to Utf8View when we change VARCHAR to utf8view